spec/hangman2_spec.rb

```diff
@@ -2,50 +2,78 @@
 
 RSpec.describe Word do
 
-  it 'returns the guesses letters' do
+  it 'returns the guessed letters' do
     word = Word.new('hello')
 
-    guessed_letters = word.guessed_letters
+    right_guess = word.right_guess
 
-    expect(guessed_letters).to eq(['_','_','_','_','_',])
+    expect(right_guess).to eq(['_','_','_','_','_',])
   end
 
   it 'returns empty letter for different length of word' do
     word = Word.new('blackberry')
 
-    guessed_letters = word.guessed_letters
+    right_guess = word.right_guess
 
-    expect(guessed_letters).to eq(['_','_','_','_','_','_','_','_','_','_'])
+    expect(right_guess).to eq(['_','_','_','_','_','_','_','_','_','_'])
   end
 
   it 'adds a letter to the current word' do
     word = Word.new('hello')
 
     word.add_guess('h')
-    expect(word.guessed_letters).to eq (['h','_','_','_','_'])
+    expect(word.right_guess).to eq (['h','_','_','_','_'])
   end
 
   it 'adds a 2nd letter to the current word' do
     word = Word.new('hello')
 
     word.add_guess('h')
     word.add_guess('l')
-    expect(word.guessed_letters).to eq (['h','_','l','l','_'])
+    expect(word.right_guess).to eq (['h','_','l','l','_'])
   end
 
   it 'adds a letter which is not matching with the current word' do
```

Comment:
Also here, are you sure you're testing this?
If you want to check that a wrong guess is added somewhere, then I should see a list with the wrong guess included in the test assertion?